domains/transfer/Transfer.API/API.IntegrationTests/Controllers/ConnectionsControllerTests.cs

```diff
@@ -148,9 +147,9 @@ public async void GetConnections_ShouldOnlyReturnConnectionsInvolvingTheCompany(
         var response = await client.GetFromJsonAsync<ConnectionsResponse>("api/connections");
 
         response.Should().NotBeNull();
-        response.Result.Should().HaveCount(2);
-        response.Result.Any(x => x.CompanyTin == ownedConnection1.CompanyBTin).Should().BeTrue();
-        response.Result.Any(x => x.CompanyTin == ownedConnection2.CompanyATin).Should().BeTrue();
+        response?.Result.Should().HaveCount(2);
```

Comment:
I think we should use `!` in tests

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/462/commits/29e0783358ba42e2edaad38de3d4221eebdc2d22